config/routes.rb

```diff
@@ -6,9 +6,11 @@
   get "up" => "rails/health#show", as: :rails_health_check
 
   # Defines the root path route ("/")
-  # root "posts#index"
+  root "users#index"
```

Comment:
Well! good try. you fix some n+1 problem. but still i notic that in the `users#index` file